arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -870,6 +870,8 @@
 		F1987D0323F7677E00DF41CC /* DisplaySubtypeSettingsViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = F1987D0223F7677E00DF41CC /* DisplaySubtypeSettingsViewController.swift */; };
```

Comment:
From the description I think the other 3 samples should be removed. But I'm not seeing these changes. So I'll probably wait for your comments rather than making changes myself. 😕